docs/source/api/v2/cdns_name_snapshot.rst

```diff
@@ -0,0 +1,542 @@
+
+	:status: The health status of this Traffic Monitor
+
+		.. seealso:: :ref:`health-proto`
+
+:stats: An object containing metadata information regarding the CDN
+
+	:CDN_name:   The name of this CDN
+	:date:       The UNIX epoch timestamp date in the Traffic Ops server's own timezone
+	:tm_host:    The :abbr:`FQDN (Fully Qualified Domain Name)` of the Traffic Ops server
+	:tm_path:    A path relative to the root of the Traffic Ops server where a request may be replaced to have this :term:`Snapshot` overwritten by the current *configured state* of the CDN
+	:tm_user:    The username of the currently logged-in user
+	:tm_version: The full version number of the Traffic Ops server, including release number, git commit hash, and supported Enterprise Linux version
+
+:trafficRouterLocations: An object containing keys which are the :ref:`names of Cache Groups <cache-group-name>` within the CDN which contain Traffic Routers
+
+	:backupLocations: An object that describes this :ref:`Cache Group's Fallbacks <cache-group-fallbacks>`
+
+		:fallbackToClosest: A string containing a boolean which defines this :ref:`Cache Group's Fallback to Closest <cache-group-fallback-to-closest>` setting; one of:
+
+			"false"
+				Do not fall back on the closest available :term:`Cache Group`
+			"true"
+				Fall back on the closest available :term:`Cache Group`
+
+	:latitude:            A floating point number that defines this :ref:`Cache Group's Latitude <cache-group-latitude>`
+	:localizationMethods: An array of strings that represents this :ref:`Cache Group's Localization Methods <cache-group-localization-methods>`
+	:longitude:           A floating point number that defines this :ref:`Cache Group's Longitude <cache-group-longitude>`
+
+.. code-block:: http
+	:caption: Response Example
+
+	HTTP/1.1 200 OK
+	Access-Control-Allow-Credentials: true
+	Access-Control-Allow-Headers: Origin, X-Requested-With, Content-Type, Accept, Set-Cookie, Cookie
+	Access-Control-Allow-Methods: POST,GET,OPTIONS,PUT,DELETE
+	Access-Control-Allow-Origin: *
+	Content-Type: application/json
+	Set-Cookie: mojolicious=...; Path=/; Expires=Mon, 18 Nov 2019 17:40:54 GMT; Max-Age=3600; HttpOnly
+	Whole-Content-Sha512: 220bc4XXwaj+s7ODd3QAF5leGj06lnApiN5E8H/B2RgxSphnQIfnwy6WWbBDjonWXPV1IWDCjBMO+rR+lAabMg==
+	X-Server-Name: traffic_ops_golang/
+	Date: Wed, 12 Dec 2018 17:36:25 GMT
+	Transfer-Encoding: chunked
+
+	{ "response": {
+		"config": {
+			"api.cache-control.max-age": "10",
+			"certificates.polling.interval": "300000",
+			"consistent.dns.routing": "true",
+			"coveragezone.polling.interval": "3600000",
+			"coveragezone.polling.url": "https://trafficops.infra.ciab.test:443/coverage-zone.json",
+			"dnssec.dynamic.response.expiration": "300s",
+			"dnssec.enabled": "false",
+			"domain_name": "mycdn.ciab.test",
+			"federationmapping.polling.interval": "60000",
+			"federationmapping.polling.url": "https://${toHostname}/internal/api/2.0/federations.json",
```

Comment:
This is using an ancient, deprecated `/internal` route which won't work in API 2.0 (firstly because they're only in Perl but that will actually get called, I think, and then the routes in Perl won't have this registered). Idk where this is being set but it should be updated to use `/api/2.0/federations` (notably with `.json` stripped out).